sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -495,7 +497,11 @@ def __init__(
 
     def _setup_transport(self):
         """Wrap the socket in an SSL object."""
-        self.sock = self._wrap_socket(self.sock, **self.sslopts)
+        try:
+            self.sock = self._wrap_socket(self.sock, **self.sslopts)
+        except FileNotFoundError as exc:
+            # TODO: invalid connection_verify, should we raise some other error?
+            raise 
```

Comment:
just a placeholder until we figure out if current behavior is correct.

@annatisch - Should invalid connection_verify result in a ConnectError?
Currently for uamqp, this scenario results in `FileNotFoundError` for the ConsumerClient and `EventHubError` for ProducerClient. Given that uamqp is raising different errors, can we say that this was a bug in our previous code? And that our current error handling for this scenario is correct?

Reply:
To clarify - what does "invalid" mean in this context?
Does it mean that value supplied was unusable, and failed client-side?
Or does it mean we attempted to use the value and it failed to connect?

Reply:
OOB disussion: keeping EventHubError and FileNotFoundError for now for parity. Creating an issue to potentially fix later here: